examples/shared_samples/network_set.rb

```diff
@@ -32,7 +40,16 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::Synergy::NetworkSet
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::NetworkSet
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::Synergy::NetworkSet
-
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::NetworkSet
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::NetworkSet
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::NetworkSet
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::Synergy::NetworkSet
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::NetworkSet
+# api_version = 1200 & variant = Synergy to OneviewSDK::API1200::Synergy::NetworkSet
+# api_version = 1600 & variant = C7000 to OneviewSDK::API1200::C7000::NetworkSet
```

Comment:
@yuvirani need to correct the Inheritance, should be OneviewSDK::API1600

Reply:
Changed to API1600